Fix/cleanup readme (#241)
Parsely/wp-parsely: README.md

## Installation

> The WP-Parsely plugin requires an active Parse.ly account. Parse.ly gives creators, marketers and developers the tools to understand content performance, prove content value, and deliver tailored content experiences that drive meaningful results.

Reviewer: ```suggestion
The plugin requires an active Parse.ly account. Parse.ly gives creators, marketers, and developers the tools to understand content performance, prove content value, and deliver tailored content experiences that drive meaningful results.
```

Author: This was taken from the main Parse.ly homepage - but I agree this comma is needed.

## Screenshots

> 1. The main settings screen of the WP-Parsely plugin  

Reviewer: The plugin isn't called `WP-Parsely`. It's called `Parse.ly`, and has a slug of `wp-parsely`. Let's aim not to mix the two.
```suggestion
1. The main settings screen of the Parse.ly plugin.  
```

![The main settings screen of the wp-parsely plugin](.wordpress-org/screenshot-1.png)

2. The settings for the Parse.ly Recommended Widget.  Engage your visitors with predictive and personalized recommendations from Parse.ly. 
![The settings for the Parse.ly Recommended Widget](.wordpress-org/screenshot-2.png)
   
3. A view of the Parse.ly Dashboard Overview. Parse.ly offers analytics that empowers you to better understand how your content is peforming.
![The Parsely Dashboard Overview](.wordpress-org/screenshot-3.png)

## Sample Parse.ly metadata

> The standard Parse.ly JavaScript tracker inserted before the closing `body` tag:
> 
>     <!-- START Parse.ly Include: Standard -->
> 
>        <script data-cfasync="false" id="parsely-cfg" data-parsely-site="wpvip.com" src="//cdn.parsely.com/keys/wpvip.com/p.js"></script>
> 
>     <!-- END Parse.ly Include: Standard -->

Reviewer: This bit isn't the metadata, so it should be moved outside of the `Sample Parse.ly metadata` section (can have its own heading).

Also:

```suggestion
The standard Parse.ly JavaScript tracker inserted before the closing `body` tag:

    <!-- START Parse.ly Include: Standard -->

       <script data-cfasync="false" id="parsely-cfg" data-parsely-site="example.com" src="https://cdn.parsely.com/keys/example.com/p.js"></script>

    <!-- END Parse.ly Include: Standard -->
```

> A sample `JSON-LD` meta tag for a home page or section page:

Reviewer: ```suggestion
A sample `JSON-LD` meta tag and structured data for a home page or section page:
```

Really, the metatag with the plugin version is separate from the JSON-LD. We may be able to remove it.

    <!-- BEGIN wp-parsely Plugin Version 2.5.0 -->
        <meta name="wp-parsely_version" id="wp-parsely_version" content="2.3"/>
            <script type="application/ld+json">
                {"@context":"http:\/\/schema.org","@type":"WebPage","headline":"WordPress VIP","url":"http:\/\/wpvip.com\/"}
            </script>
    <!-- END wp-parsely Plugin Version 2.5.0 -->

> A sample `JSON-LD` meta tag for an article or post:

Reviewer: ```suggestion
A sample `JSON-LD` structured data element for an article or post:
```
